report/main.qmd

```diff
@@ -0,0 +1,103 @@
+  text: |
+    \usepackage{authblk}
+    \pagenumbering{gobble}
+---
+
+{{< pagebreak >}}
+
+```{=tex}
+\pagenumbering{arabic}
+```
+
+# Introduction
+
+Strong evidence has linked habitual intake of sugar-sweetened beverages (SSBs) with weight gain and a higher risk of type 2 diabetes mellitus, cardiovascular diseases, and even cancer @Malik2022. Multiple policy interventions to inhibit consumption of SSBs have been proposed @Krieger2021, including taxation of SSBs, limiting access to SSBs in schools and healthcare facilities, and adding informative package labelling. As policymakers create potential interventions to limit SSB consumption, there is a need for data-driven methods of determining their efficacy.
+
+This study aims to evaluate five interventions which intend to shift consumer behaviour towards zero-calorie beverages. These interventions include three information-based interventions and two financial interventions.
+
+The main statistical question is whether these interventions are associated with differences in the average proportion of daily sales of zero-calorie beverages. Secondary questions include whether there are significant interactions between interventions and site effects, and whether a combination of interventions was more effective than the individual interventions.
+
+# Data Description and Summaries
+
+The data come from an interrupted quasi-experimental study on sales of sweetened and zero-calorie beverages which was conducted in three United States hospitals over a 30-week period. Each observation represents the total daily sales of bottled beverages across multiple storefronts in a single site. Sales were recorded automatically by point of sale terminals in Site A, and manually by cashiers at Sites B and C. Sales were recorded every day from the start of the experiment until the final day, including weekends and holidays.
+
+The following variables were collected: the day of the experiment (e.g. Day 1), the corresponding weekday, the site, and the ongoing intervention. The day counter is a positive integer. Categorical variables include weekday with seven levels, site with three levels, and intervention with nine levels. The daily sales metrics include the number of bottled zero-calorie and sweetened beverages sold per site. These are each non-negative integer variables. The data also includes the total number of beverages sold per site, which is another non-negative integer value.
+
+Some observations additionally include the sales of 100% juice, orange juice, and sports drinks, which are recorded as integers. However, these counts are only consistently recorded at Site A, and rarely recorded for Site B and C. There are nine days which are missing counts for the bottled zero-calorie and sugared beverages. Each of these nine observations contains missing data for every sales-related column. Of these nine observations, seven occur in the final week of the study at Site B, and two occur mid-study at Site C.
```

Comment:
This sentence "There are nine days which are missing counts for the bottled zero-calorie and sugared beverages" is a little confusing. Maybe rephrase?